src/spaceObjects/artifact.h

```diff
@@ -11,6 +11,11 @@ class Artifact : public SpaceObject, public Updatable
     float artifact_spin=0.0;
     bool allow_pickup;
     ScriptSimpleCallback on_pickup_callback;
+
+    string radar_trace_icon="RadarBlip.png";
+    float radar_trace_scale=0;
```

Comment:
default of zero?

Reply:
Yep, I used the value of zero to set the standard scaling behavior. See artifact.cpp line 84

Reply:
ah, missed that :)